storageReader_linux.go

```diff
@@ -0,0 +1,18 @@
+package loge
+
+import (
+	"syscall"
+)
+
+func getStoragePercent(logDir string) float64 {
+	var fileSystemStats syscall.Statfs_t
+	if err := syscall.Statfs(logDir, &fileSystemStats); err != nil {
+		return -1
+	}
+
+	totalBytes := float64(fileSystemStats.Blocks * uint64(fileSystemStats.Bsize))
+	usedBytes := float64((fileSystemStats.Blocks - fileSystemStats.Bavail) * uint64(fileSystemStats.Bsize))
+
+	percentUsage := (usedBytes / totalBytes)
+	return percentUsage
+}
```

Comment:
Are the Linux and Darwin implementations identical? If so, we can probably condense.